### PR TITLE
Improved List of Independence Day of Nations

### PR DESCRIPTION
**Pull request by RCMainak**
Add few(10) of the missing countries. Improved json format for 2 nations. Added 2 nation Aliases.
Addressed issue #2790.

References used : 

- https://en.wikipedia.org/wiki/List_of_countries_and_dependencies_and_their_capitals_in_native_languages

- https://en.wikipedia.org/wiki/List_of_national_independence_days

- http://geography.about.com/od/lists/a/independenceday.htm

---
IA Page : https://duck.co/ia/view/independence_day
Maintainer: @jarmokivekas

### CODE REVIEW

**Comment by daxtheduck** [comment]
@RCMainak Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/independence_day/independence_days.json [hunk 1]
```diff
@@ -157,9 +163,11 @@
     "tonga":  [{"date": "June 4th", "year": "1970"}],
     "trinidad and tobago":  [{"date": "August 31th", "year": "1962"}],
     "tunisia":  [{"date": "March 20th", "year": "1956"}],
+    "Turkey":  [{"date": "October 29th", "year": "1923"}],
```
Comment (moollaza) [inline share/goodie/independence_day/independence_days.json]: looks like we used lowercase letters everywhere else?

File: share/goodie/independence_day/independence_days.json [hunk 2]
```diff
@@ -37,6 +39,7 @@
     "comoros":  [{"date": "July 6th", "year": "1975"}],
     "democratic republic of the congo":  [{"date": "June 30th", "year": "1960"}],
     "republic of the congo":  [{"date": "August 15th", "year": "1960"}],
+    "Cook Islands":  [{"date": "August 4th", "year": "1965"}],
```
Comment (moollaza) [inline share/goodie/independence_day/independence_days.json]: lowercase here too

File: share/goodie/independence_day/independence_days.json [hunk 3]
```diff
@@ -80,14 +83,14 @@
     "jordan":  [{"date": "May 25th", "year": "1946"}],
     "kazakhstan":  [{"date": "December 16th", "year": "1991"}],
     "kenya":  [{"date": "December 12th", "year": "1963"}],
+    "Kiribati":  [{"date": "July 12th", "year": "1979"}],
```
Comment (moollaza) [inline share/goodie/independence_day/independence_days.json]: lowercase here too

**Comment by RCMainak** [comment]
OpS! My bad... Side effect of copy paste... :tongue:

**Comment by RCMainak** [comment]
Sorry, I made changes to some other goodies. I have undone all the changes it should not affect anything. other than Independence day goodies

**Comment by moollaza** [comment]
> Sorry, I made changes to some other goodies. I have undone all the changes it should not affect anything other than Independence day goodies

No problem, make sure you don't use your master branch for any pull requests in the future. You should use a unique branch for each PR so that changes are always separated :+1:

**Comment by moollaza** [comment]
@RCMainak sorry I just realized that your PR is different than https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2791 which came first.

Can you and @t2013anurag work together and put all the changes into 1 PR? That will simplify things :+1:

**Comment by RCMainak** [comment]
@moollaza Well, I thought the same thing in & thought collaboration will be hard to do because of the communication gap in GitHub. So, I shamelessly copied all his changes and included them in mine.

P. S. few of those changes were already in my list as far as I can recall I copied exactly one nation alias that I somehow missed. I done bit of research before filling the issue, just wasn't sure how to add those.

**Comment by mintsoft** [comment]
@RCMainak I'm closing this as #2720 came first and the changes have been made identical in both now; it'll be easier to just have 1 PR in progress.

**Comment by RCMainak** [comment]
@mintsoft I think you made some mistake #2720 have nothing to do with this one...

**Comment by mintsoft** [comment]
@RCMainak You are correct I meant #2791; Thanks

**Comment by alohaas** [comment]
@RCMainak, thanks for trying to improve the list of Independence Day Nations IA, sorry it wasn't merged, but you did a great job and we really appreciate your work to maintain this IA. We'd like you to help our community tackle the next big challenge – our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements 

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by RCMainak** [comment]
@alohaas thank you for the kind words :smiley: I will try to do some more work :tongue:

**Pull request closed without merging**